src/ILVerification/src/ILVerifyTypeSystemContext.cs

```diff
@@ -102,7 +102,7 @@ internal EcmaModule GetModule(PEReader peReader)
                 return existingModule;
```

Comment:
Do we need to check here that the `existingModule.Assembly` matches the requested `containingAssembly`?